x

```diff
@@ -29,5 +29,12 @@ for SEARCH_PYTHON in py python3 python python2; do
         exec "$python" $extra_arg "$xpy" "$@"
     fi
 done
+
+for SEARCH_PYTHON in (compgen -c "python"); do
```

Comment:
Why is this a for loop? The powershell script is using the first one returned, which makes sense to me since `exec` never returns.